### DISCUSSION TITLE
Browser transparency

### DISCUSSION THREAD

**Discussion by NikitaPZ** [Naezr/ShyFox#148] [Ideas]
Is there any way to add a bit of transparency to the browser background color? Maybe even add an option for it.

**Comment by Naezr** [comment]
This is definitely cool, but there are more important things (basic functionality) in development for ShyFox 4.0 right now. Version 3.8.1 is kind of frozen now, I will only release critical fixes for it.

**Reply by NikitaPZ** [reply]
Sounds about right. Maybe just an idea for later then.
